web-jehlomat/src/assets/images/profile-data.svg

```diff
@@ -0,0 +1,34 @@
+<svg width="608" height="400" viewBox="0 0 608 400" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
I believe we already have this image in the project.

Reply:
yes, you are now... after pulling latest code I can see that we have it, will remove the one I added